src/components/input/ClearButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC, MutableRefObject } from 'react';
+
+import { CloseIcon } from '../icons';
+
+type Props = {
+  inputRef: MutableRefObject<HTMLInputElement | null>;
+};
+
+const triggerNativeEventFor = (
+  elm: Element,
+  { event, ...valueObj }: Record<string, unknown> & { event: Event['type'] },
+) => {
+  if (!(elm instanceof Element)) {
+    throw new Error(`Expected an Element but received ${elm} instead!`);
```

Comment:
should we really throw? Wouldn't that cause a client side error page ?

Reply:
We don't necessarily have to throw, but it would result in an unexpected behaviour if we would want to trigger an event for something else than a DOM element. The way that events are handled I think this would result in an error message in the console and not a FE error page